src/main/java/blusunrize/immersiveengineering/common/blocks/metal/DieselGeneratorBlockEntity.java

```diff
@@ -158,32 +161,37 @@ public void tickServer()
 
 		if(!isRSDisabled()&&!tanks[0].getFluid().isEmpty())
 		{
-			GeneratorFuel recipe = recipeGetter.apply(level, tanks[0].getFluid().getFluid());
-			if(recipe!=null)
+			int output = IEServerConfig.MACHINES.dieselGen_output.get();
+			List<IEnergyStorage> presentOutputs = outputs.stream()
+					.map(CapabilityReference::getNullable)
+					.filter(Objects::nonNull)
+					.collect(Collectors.toList());
+			if(!presentOutputs.isEmpty()&&EnergyHelper.distributeFlux(presentOutputs, output, false) < output)
 			{
-				int burnTime = recipe.getBurnTime();
-				int fluidConsumed = FluidAttributes.BUCKET_VOLUME/burnTime;
-				int output = IEServerConfig.MACHINES.dieselGen_output.get();
-				List<IEnergyStorage> presentOutputs = outputs.stream()
-						.map(CapabilityReference::getNullable)
-						.filter(Objects::nonNull)
-						.collect(Collectors.toList());
-				if(!presentOutputs.isEmpty()&&
-						tanks[0].getFluidAmount() >= fluidConsumed&&
-						// Sort receivers by lowest input
-						EnergyHelper.distributeFlux(presentOutputs, output, false) < output)
+				consumeTick--;
+			}
```

Comment:
For single-line conditionals we usually skip the braces, but I can ignore that =p

Reply:
I was making absolutely sure I followed braces codestyle after the issues I'd had :p